ext/iperf3/Kconfig

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# iperf3 NRF integration
+
+config  NRF_IPERF3
+	bool "Iperf3 NRF integration"
+	depends on POSIX_API && !PTHREAD_IPC && !NET_SOCKETS_POSIX_NAMES && NEWLIB_LIBC
+	imply NEWLIB_LIBC_FLOAT_PRINTF
+	help
+	  Enable Iperf3 NRF integration
+
+if NRF_IPERF3
+menu "NRF Iperf3 selections"
+
+config NRF_IPERF3_HOST_NAME
+	string "Host name for iperf3 to be sent as a part of cookie"
+	default "nrf9160"
+
+config NRF_IPERF3_RESULTS_WAIT_TIME
+	int "Time in seconds to be waited to send and recv results"
+	default 15
+
+config NRF_IPERF3_CLIENT_TEST_START_TIME
+	int "Time in seconds to be waited after ctrl socket has been created to actual test start"
```

Comment:
Please shorten and use help text for detailed explanation.

Reply:
Fixed